internal/cmd/manager/instance/run/lifecycle/run.go

```diff
@@ -67,12 +73,13 @@ func (i *PostgresLifecycle) runPostgresAndWait(ctx context.Context) <-chan error
 		}
 
 		// once the database will be up we'll connect and setup everything required
-		err = configureInstancePermissions(ctx, i.instance)
-		if err != nil {
-			contextLogger.Error(err, "Unable to update PostgreSQL roles and permissions")
-			errChan <- err
-			return
-		}
+		go func() {
```

Comment:
I worry about this being a goroutine without a waitgroup or similar.
Seems if there is an error here but it doesn't happen super quick, we could ignore it.